recipe/meta.yaml

```diff
@@ -154,26 +188,28 @@ outputs:
         - sysroot_{{ ctng_target_platform }}
       run:
         # For cpp and crt{i,n}.o
-        - gcc_impl_{{ ctng_target_platform }} >=7.2.0
-        - {{ pin_subpackage("libgfortran-ng") }}
-        - {{ pin_subpackage("libgcc-ng", max_pin=None) }}
+        - gcc_impl_{{ ctng_target_platform }} >={{ ctng_gcc }}
+        - libgfortran-ng {{ ctng_gcc }}.*       # [target_platform == ctng_target_platform]
         - sysroot_{{ ctng_target_platform }}
+        # Any version newer than the devtoolset works. This can be removed when bootstrapping.
```

Comment:
I have the same two questions here as I did for gcc above.

Reply:
Same as before